### PR TITLE
paginator: use universal regexp to support some edge cases

### PR DESCRIPTION
**Pull request by vrutkovs**
{{pytest: cfme/tests/configure/test_visual_cloud.py -v --long-running }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 895a15fcf224694ec6f0bcc8e0c74ff48b0d89ea:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/web_ui/paginator.py [hunk 1]
```diff
@@ -16,9 +16,8 @@
 _page_cell = '//td//td[contains(., " of ")]|//li//span[contains(., " of ")]'
 _check_all = Input("masterToggle")
 
-_prefix = r"(?:Item|Items|Rows|Row)*\s*"
-_regexp = {version.LOWEST: r"{}\((?P<first>\d+)-(?P<last>\d+) of (?P<total>\d+)\)".format(_prefix),
-           "5.5": r"Showing (?P<first>\d+)-(?P<last>\d+) of (?P<total>\d+) (?:item|items?)?"}
+_prefix = r"(?:Item|Items|Rows|Row|Showing)*\s*"
```
Comment (mfalesni) [inline cfme/web_ui/paginator.py]: Actually shouldn't it be rather `?` than `*` as I am looking at it now? btw you can DRY here, `Items?|Rows?|Showing`.

**Comment by dajoRH** [comment]
Lint report for commit 0dbf8c8e73cbb045443b2f78119f97c5ba0f049f:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7b75504f99ae7188a0ca3606f7e19942476a57b8:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1e1b7d5bfb29d17e3d68140abd49bf3a87047949:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*
